app/views/trip/new.html.erb

```diff
@@ -1,13 +1,18 @@
 <h4><%=@location %></h4>
-<h4>Climbing route options for your Trip!</h4>
-<% climb_results.each do |climb| %>
-  <section class="climbs">
-    <h5><%= climb.name %></h5>
-    <p><%= climb.climb_type %></p>
-    <p><%= climb.rating %></p>
-    <p><%= climb.stars %></p>
-    <p><%= climb.pitches %></p>
-    <p><%= climb.location %></p>
-    <p><%= climb.url %></p>
-  </section>
-<% end %>
\ No newline at end of file
+<% if climb_results.count > 0 %>
+  <h4>Climbing route options for your Trip!</h4>
+  <% climb_results.each do |climb| %>
+    <section class="climbs">
+      <h5><%= climb.name %></h5>
+      <p><%= climb.climb_type %></p>
+      <p><%= climb.rating %></p>
+      <p><%= climb.stars %></p>
+      <p><%= climb.pitches %></p>
+      <p><%= climb.location %></p>
+      <p><%= climb.url %></p>
+    </section>
+  <% end %>
+<% else %>
+  <h4>No climbs exist for those parameters.</h4>
+  <%= button_to "Try Again", "/search/climbs/new", method: :get %>
```

Comment:
Great sad path implementation here for scenarios resulting in receiving no results.